lib/travis/api/v3/models/env_var.rb

```diff
@@ -2,7 +2,7 @@ module Travis::API::V3
   class Models::EnvVar < Travis::Settings::Model
     attribute :id, Integer
     attribute :name, String
-    attribute :value, String
+    attribute :value, Travis::Settings::EncryptedValue
```

Comment:
User and Token use this pattern with the EncryptedColumn V3 extension code, instead of the Travis::Settings stuff. Just an FYI that we have 2 different patterns for doing encryption in V3 with adding this code. The other pattern is this:
``` ruby
serialize         :token, Extensions::EncryptedColumn.new(disable: true)
```